src/app/pipes/capitalize-string.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { CapitalizeString } from '@app/pipes/capitalize-string';
+
+describe('CapitalizeString', () => {
+  const pipe = new CapitalizeString();
+  const stringToCapitalize = 'test';
+
+  it('create an instance', () => {
+    expect(pipe).toBeTruthy();
+  });
+
+  it('should return capitalized word', () => {
+    const res = pipe.transform(stringToCapitalize);
+    expect(res).toEqual('Test');
```

Comment:
Same for this one i think we could do one more test that checks the else of ternary operator, as it seems that if we give something 'empty' to the pipe, it returns ''